doc/man7/OSSL_PROVIDER-base.pod

```diff
@@ -57,25 +57,90 @@ currently permitted.
 
 The OpenSSL base provider supports these operations and algorithms:
 
+=head2 Random Number Generation
+
+The default provider also includes the "SEED-SRC" algorithm.
```

Comment:
s/default/base/ ?

Reply:
Apparently @romen is confused similarly as myself. @mattcaswell It really needs to be moved after the list and perhaps even reformulated? `The "SEED-SRC" algorithm is also included in the default provider.`